RecoLocalCalo/HGCalRecAlgos/interface/HGCalRecHitSimpleAlgo.h

```diff
@@ -41,7 +41,8 @@ class HGCalRecHitSimpleAlgo : public HGCalRecHitAbsAlgo {
     //    float clockToNsConstant = 25;
     float energy = uncalibRH.amplitude() * adcToGeVConstant_;
     float time   = uncalibRH.jitter();
-    if(time<0) time   = 0; // fast-track digi conversion
+
+    //if(time<0) time   = 0; // fast-track digi conversion
```

Comment:
is the range of values going to be handled in HGCRecHit constructor (not sure if this reset of negative values to zero implies anything on value conversion)